src/test/java/edu/kis/powp/jobs2d/events/SelectTestFigureOptionListener.java

```diff
@@ -22,6 +22,14 @@ public void actionPerformed(ActionEvent e) {
 			case "Figure Joe 2":
 				FiguresJoe.figureScript2(driverManager.getCurrentDriver());
 				break;
+			case "Triangle":
+				DriverCommand triangle = FiguresFactory.createTriangle();
+				triangle.execute(driverManager.getCurrentDriver());
+				break;
+			case "Parallelogram":
```

Comment:
Menu action should not depend on its caption. Enum or separate listeners would be better. At least you should consider constants.